### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by nnyyxxxx**
## Type of Change
- [x] New feature
- [x] Documentation Update
- [x] Refactoring

## Description
Removed unneeded image and replaced it with the primary preview, added the CONTRIBUTING.md contents to the contribute section, added a starting point for the user guide, added a starting point for the known issues section.

## Checklist
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,14 +3,6 @@
...
 ![GitHub Downloads (specific asset, all releases)](https://img.shields.io/github/downloads/ChrisTitusTech/linutil/start.sh?label=Total%20Downloads&style=for-the-badge)
 
-## Running latest release of LinUtil
-
-* You will first need to start a Terminal on your linux machine.
-* Now you will need to run following command:
-   ```
-   curl -fsSL https://christitus.com/linux | sh
-   ```
-
```
Comment (Real-MullaC) [inline docs/index.md:13]: This should be kept so people can see how to run the script.

**Review by Real-MullaC — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (nnyyxxxx) [reply docs/index.md:13]: in the description section of this PR I gave a reason on why I removed it

Comment (Real-MullaC) [reply docs/index.md:13]: We have it like this how it is in this docs on WinUtil.

Comment (nnyyxxxx) [reply docs/index.md:13]: I'll have a look, and do some changes then.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**